uuid/uuid.go

```diff
@@ -4,7 +4,7 @@
 //
 // Since: 1.6.0.
 //
-// See also
+// # See also
```

Comment:
How does it looks when doc is compiled?

Reply:
Like a subtitle:

![godoc](https://user-images.githubusercontent.com/29621138/183143697-d1ba72af-e87c-4576-8b93-7411f4dfb639.png)

See `Doc Comments` on https://tip.golang.org/doc/go1.19